types/empty-dir/index.d.ts

```diff
@@ -9,19 +9,19 @@
 export = emptyDir;
 
 declare function emptyDir(
-    dir: string,
+    dir: string | string[],
```

Comment:
Please use `readonly` arrays for inputs that are not modified by a function:
```suggestion
    dir: string | readonly string[],
```